Kickstarter-iOS/Views/Storyboards/SettingsFooterView.xib

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="14460.31" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="Vl0-YN-TOM" customClass="SettingsFooterView" customModule="Kickstarter_Framework" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+            <subviews>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="fnW-tg-iSJ">
+                    <rect key="frame" x="12" y="12" width="351" height="643"/>
+                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                    <nil key="textColor"/>
+                    <nil key="highlightedColor"/>
+                </label>
+            </subviews>
+            <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+            <constraints>
+                <constraint firstAttribute="bottom" secondItem="fnW-tg-iSJ" secondAttribute="bottom" constant="12" id="CUo-3i-EFj"/>
+                <constraint firstItem="fnW-tg-iSJ" firstAttribute="leading" secondItem="Vl0-YN-TOM" secondAttribute="leading" constant="12" id="WPU-br-A3U"/>
```

Comment:
Can we constraint against the superview's `margins` and set the `layoutMargins` directly on the `SettingsFooterView`?

Reply:
I've tried to reflect the layout & logic of `SettingsHeaderView` as closely as possible for now. For consistency, I'd suggest these two (header & footer) use the same functionality (so either keep constants or also update header to use `layoutMargins`.

One thing to mention is that we will also have `SettingsGroupedHeaderView` & `SettingsGroupedFooterView` coming out of the `Create Password` feature branch that should support dynamic type and are fully created programatically. We've already discussed this with Nino and it means we'll have to start thinking about how we remove one or other (for limiting the amount of duplicate functionality).

Reply:
Okay makes sense 👍 I'd lean towards using the programmatic header/footer views and deleting these ones

Reply:
Sounds good..I'll keep these changes for now and try to figure out our strategy for when we have programmatic headers footers available post `Create Password` feature release / merge.